.elasticbeanstalk/config.yml

```diff
@@ -0,0 +1,9 @@
+branch-defaults:
+  master:
+    environment: mnl-api
+global:
+  application_name: mnl
+  default_ec2_keyname: XXXXXX
+  default_platform: 64bit Debian jessie v2.8.4 running Python 3.4 (Preconfigured - Docker)
```

Comment:
Can't remember off the top of my head if this matches but is there a way to make sure we keep this and the django/Dockerfile in sync somehow?

Reply:
in sync how?  i have a hard time seeing how they're similar

Reply:
I'm not sure it matters, not exactly sure how you're setting up the prod environment.

I was thinking about how the django service runs on python:3.6-alpine https://github.com/monday-night-lights/mnl-api/blob/master/django/Dockerfile#L1

Reply:
Will we use Docker in prod? Could we bundle everything into an image? 

We wouldn't need Python on the server at all at that point.

Reply:
Ahh. I thought you meant automatically in-sync, not having the same docker image.  I think I misunderstood you.  I'll look into it